src/OrchardCore.Themes/TheComingSoonTheme/Assets.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "generateSourceMaps": false,
+    "inputs": [ "wwwroot/scss/bootstrap-oc.scss" ],
+    "output": "wwwroot/css/bootstrap-oc.css"
+  }
+]
```

Comment:
We don't want to build themes on every `gulp build` this is why you need to run it directly on each wwwroot folder of each of these themes. At least for now keep it but add an extra line at the end of file.